### ISSUE TITLE
Fix domain list entry on backend

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/gpc-privacy-choice#218]
As we discussed earlier today, the domain list is not being populated per below.

<img width="278" alt="Screen Shot 2022-05-31 at 9 47 42 AM" src="https://user-images.githubusercontent.com/11034003/171306253-febc40c3-9727-4385-97fa-ae5a6aea78e0.png">

There are two potential solutions to this issue:
1. Fix it such that it is being populated
2. Just remove the functionality (UI and logic)

The first is preferred. However, more important than which of the two solutions we pick is to do it quickly. We can get the domains after the fact without the domain list just from the browsing history. So, by the end of the week we want to have everything ready.

@bella-tassone will work on this and @ekuller and @chunyuema will assist and review as necessary.

**Comment by bella-tassone** [comment]
I've been working on this issue for the past couple days, and while it is mostly functional, it has one small quirk. If the user hits the snooze button on a banner scheme, the site on which they hit the snooze button will be present in the domain list, although they didn't make a privacy choice for it. I understand this to be an async issue (the mute bool doesn't update in time), but I've been struggling with how to get around it. Personally, I don't think it's that big of an issue, but I thought I'd document the problem before making a pull request/ditching the domain list component of the data.

**Comment by SebastianZimmeck** [comment]
OK, thanks, @bella-tassone! I think that is a small quirk that we do not need to worry about. So, feel free to close this issue once the code is merged into main.
